README.md

```diff
@@ -1,52 +1,29 @@
 # Media Metadata
-Place this app in **owncloud/apps/**
+---
 
-## Building the app
+## Description
+A Cloud Application, which provides CRUD access to the metadata stored in the images. 
```

Comment:
A cloud application, which provides CRUD access to the metadata stored in media files.